keyboards/cocoa40/keymaps/default/config.h

```diff
@@ -0,0 +1,22 @@
+/* Copyright 2019 'Naoto Takai'
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+// place overrides here
+
+#define TAPPING_TERM 200
```

Comment:
Ideally the defaults should be used for default keymaps where possible, `TAPPING_TERM` defaults to 200 anyway so this can be safely removed.
```suggestion
```